wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/CommandScheduler.java

```diff
@@ -530,6 +530,13 @@ public void enable() {
     m_disabled = false;
   }
 
+  /**
+   * Prints list of epochs added so far and their times.
+   */
+  public void printWatchdogEpoch() {
```

Comment:
Make this plural.

Reply:
oops, done

Reply:
It still says `Epoch` for me on GitHub, did you push?